Assets/Scripts/SketchControlsScript.cs

```diff
@@ -146,6 +146,7 @@ public enum GlobalCommands
             RenameSketch = 5200,
             OpenLayerOptionsPopup = 5201,
             RenameLayer = 5202,
+            LanguagePopup = 5203,
```

Comment:
Be careful doing this. Although "Rename Layer" is merged, I sometimes use the same block of enums for PRs with closely related functionality. It's best to grab your own block of numbers: https://docs.google.com/spreadsheets/d/17OOrxFgrA8COwxAinj9e7r2zwcKxLP4J306c5woZooc/edit#gid=0